android/app/src/main/java/org/fossasia/openevent/utils/ShowNotificationSnackBar.java

```diff
@@ -56,6 +57,11 @@ public void onClick(View v) {
 
 
     public void buildNotification(){
+        ConnectivityManager connectivityManager = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
+        android.net.NetworkInfo isWifi = connectivityManager.getNetworkInfo(ConnectivityManager.TYPE_WIFI);
+        if (!isWifi.isConnectedOrConnecting()){
+            return;
```

Comment:
Can you make it true and put the code to display it in the if conditional statement pls, using return helps but putting the code reduces a line :) and improves the code viewing standard, else good work :)